ga4gh/backend.py

```diff
@@ -41,11 +41,27 @@ def _parsePageToken(pageToken, numValues):
 
 def _getVariantSet(request, variantSetIdMap):
     variantSetId = request.variantSetId
+    variantSet = _safeMapQuery(
+        variantSetIdMap, variantSetId,
+        exceptionClass=exceptions.VariantSetNotFoundException)
+    return variantSet
+
+
+def _safeMapQuery(idMap, id_, exceptionClass=None, idErrorString=None):
+    """
+    Attempt to retrieve a value from a map, throw an appropriate error
+    if the key is not present
+    (works for both dict and list types)
```

Comment:
Yes, but we're explicitly saying it's a map, and passing a key, so this is just confusing. Anyway, a list doesn't throw a KeyError if an index doesn't exist (an IndexError) so this wouldn't work as advertised.

Just delete the comment --- we don't need it to support lists.